assets/src/edit-story/components/inspector/document/test/slug.js

```diff
@@ -60,4 +67,38 @@ describe('SlugPanel', () => {
     const url = getByRole('link', { name: 'https://example.com/foo' });
     expect(url).toBeDefined();
   });
+
+  it('should not display permalink', async () => {
+    const { getByAriaLabel, updateStory } = setupPanel();
+    const input = getByAriaLabel('Edit: URL slug');
```

Comment:
Similarly here, change to `getByRole`. An input field has implicit `role="textbox"`.